packages/liveblocks-react-ui/src/components/AiChat.tsx

```diff
@@ -207,7 +209,7 @@ export const AiChat = forwardRef<HTMLDivElement, AiChatProps>(
               {$.AI_CHAT_MESSAGES_ERROR(error)}
             </div>
           ) : messages.length === 0 ? (
-            <Empty />
+            <Empty chatId={chatId} copilotId={copilotId} />
```

Comment:
Passing `chatId` and `copilotId` to the `Empty` component so that users don't have to drill these props themselves.